backend/src/controllers/saveForMeController/saveLinks/index.js

```diff
@@ -0,0 +1,33 @@
+const { BodyPropertyError } = require('errors-stack')
+const logger = require('../../../services/logger')
+const { errorHandler } = require('../../../helpers')
+const { saveForMeService } = require('../../../services')
+
+/**
+ * @param {*} res
+ * @param {*} req
+ * @return {Promise<any>}
+ */
+const saveLinks = async (req, res) => {
+  const { email, data, service } = req.body
+  try {
+    logger.print({
+      severity: 'info',
+      message: `Initiation of service: ${service} with follow data: ${logger.beautify(data)} `,
```

Comment:
```suggestion
      message: `Initiation of service: ${service} with follow data: ${logger.beautify(data)}`,
```

Coisa boba d mais kkkk